src/logger.js

```diff
@@ -38,6 +40,30 @@ class Logger {
   }
 }
 
+class StreamForLogger {
+  constructor(folderPath) {
+    this.folderPath = folderPath;
+    this.date = new Date().toISOString().substring(0, 10); 
+    this.#createFileStream();
+  }
+  write(msg) {
+    process.stdout.write(msg);
+    const currentDate = new Date().toISOString().substring(0, 10);
+    if (currentDate !== this.date) {
+      this.date = currentDate;
+      this.logFileStream.end();
+      this.#createFileStream();
+    }
+    this.logFileStream.write(msg);
+  }
+
+  #createFileStream() {
+    const filePath = path.join(this.folderPath, `${this.date}.log`);
+    this.logFileStream = fs.createWriteStream(filePath, { flags: 'a' });
```

Comment:
Make the property private and be sure to always define all the properties of a class in the constructor.

Reply:
Or Prop list.

Reply:
Done